src/styles/_reset.scss

```
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```

```suggestion
body {
```
I think, it`ll be enough)